### DISCUSSION TITLE
Events deleted directly after clicking 'Add Policy' (retention)

### DISCUSSION THREAD

**Discussion by wardboumans** [datalust/seq-tickets#1287] [Help] [answered]
Even without clicking 'Save changes', my events are gone. I went to check on free disk size to determine a policy and the DB was already just 5MB.

Nice one...

Seq 5.1.3364
Single-User License

**Comment by wardboumans** [comment, marked as answer]
Seems I was looking in the wrong data folder. I setup a custom location on initial install (the directory is there), but somehow it started logging to the ProgramData folder. Maybe after an update.

**Reply by nblumhardt** [reply]
Thanks for the follow-up. If you need a hand getting it pointed back to the right location, please drop us a line 👍
